Makefile

```diff
@@ -50,19 +50,23 @@ fmt:	go-fmt update-licenses
 
 .PHONY: go-fmt
 go-fmt:
-	$(GO) fmt $(PKG)
+	$(GO) fmt -l -s -w $(PKG)
 
 .PHONY: vet
 vet:
 	$(GO) vet $(PKG)
 
+golangci-lint-install:
+	@scripts/install-tools.sh --golangci
+	
+.PHONY: go-lint
+go-lint: golangci-lint-install
+	$(GOLANGCI_LINT) config verify
```

Comment:
I don't think this will work when the script has installed golangci-lint under .local/bin. I think following something similar to what we do for installing [protoc](https://github.com/apple/container-builder-shim/blob/1941038b3b833cc6e7f030c36427a8c1cad24e8d/Protobuf.Makefile#L42) would be good. What do you think?